src/index.js

```diff
@@ -1 +1,67 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import getCardTypes from './modules/getCardTypes.js';
+import shorterArray from './modules/shorterArray.js';
+
+const cardBoxes = document.querySelector('.card-boxes');
+
+// getCardImages
+const pokemonImage = (pokeID, container) => {
+  const image = document.createElement('img');
+  // imageID
+  image.srcset = `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokeID}.png`;
+  container.append(image);
+};
+
+// renderFetchedData
+const renderPokemonData = (pokeData) => {
+  const pokeCard = document.createElement('div');
+  pokeCard.classList.add('poke-dex');
+  // console.log(pokeData.moves);
+  pokemonImage(pokeData.id, pokeCard);
+
+  const pokemonName = document.createElement('h2');
+  pokemonName.innerText = pokeData.name;
+  pokeCard.appendChild(pokemonName);
+
+  const moves = shorterArray(pokeData.moves);
+  console.log(moves);
+  const pokeType = document.createElement('ul');
+  pokeCard.appendChild(pokeType);
+
+  getCardTypes(pokeData.types, pokeType);
+
+  pokeCard.innerHTML += `
+      <p>#${pokeData.id}</p>
+      `;
+
+  cardBoxes.appendChild(pokeCard);
+};
+
+// getMoreCardInfo;
+const getMoreInfo = (pokemon) => {
+  const sourceUrl = pokemon.url;
+  fetch(sourceUrl)
+    .then((response) => response.json())
+    .then((pokeData) => {
+      renderPokemonData(pokeData);
+    })
+    .catch((error) => {
+      console.error('Error fetching more info:', error);
```

Comment:
I think it would be best to remove this console.log and replace it with a return. It allows us to follow our JS best practices